mm/2s2h/Enhancements/Cutscenes/MiscInteractions/SkipDekuTelescope.cpp

```diff
@@ -0,0 +1,118 @@
+#include <libultraship/bridge.h>
+#include "2s2h/GameInteractor/GameInteractor.h"
+#include "2s2h/ShipInit.hpp"
+
+extern "C" {
+#include "variables.h"
+
+#include "overlays/actors/ovl_En_Scopenuts/z_en_scopenuts.h"
+void func_80BCB078(EnScopenuts* enScopenuts, PlayState* play);
+
+void func_80BCB1C8(EnScopenuts* enScopenuts, PlayState* play);
+s32 func_80BCC2AC(EnScopenuts* enScopenuts, Path* path, s32 arg2_);
+f32 func_80BCC448(Path* path, s32 arg1, Vec3f* arg2, Vec3s* arg3);
+}
+
+#define CVAR_NAME "gEnhancements.Cutscenes.SkipMiscInteractions"
+#define CVAR CVarGetInteger(CVAR_NAME, 0)
+
+// has reached point
+s32 modified_func_80BCC2AC(EnScopenuts* enScopenuts, Path* path, s32 arg2_) {
+    Vec3s* points = (Vec3s*)Lib_SegmentedToVirtual(path->points);
```

Comment:
iirc this doesn't actually do anything
```suggestion
    Vec3s* points = path->points;
```